src/start.ts

```diff
@@ -0,0 +1,5 @@
+import { startBot } from "./bot";
+import { startServer } from "./server";
+
+startBot();
+startServer();
```

Comment:
If I get this right, the `yarn start` script will start both the bot and the server in the same instance?

And the bot doesn't require the server anymore?

Also, minor question but just for convention, wouldn't `index.ts` be a better name for the entry script?

Reply:
> If I get this right, the yarn start script will start both the bot and the server in the same instance?

Correct.

> And the bot doesn't require the server anymore?

Yes, but we still probably want to run both - for one reason because the server exposes the prometheus metrics, healthcheck etc.

> Also, minor question but just for convention, wouldn't `index.ts` be a better name for the entry script?

I couldn't use `index.ts` in `{server,bot}/index.ts` because in there I have to have two files - one that starts, one that only exports a function without running it. (one file used to run the single component as en entrypoint, the other file to import the function and run both components)

Reply:
I guess I could have a single `src/{server,bot}/index.ts` as an entrypoint, and then in `src/index.ts` do:

```
import 'server/index.ts'
import 'bot/index.ts'
```

That would fire those function without the need to import a function and invoke it.

Reply:
If we follow this Rfc - 
https://github.com/paritytech/opstooling/blob/jp/rfc-modules-organization/rfc-modules-organization.md#module-patterns
then we should avoid using index.ts, however we always can revisit it